aws_lambda_builders/workflows/rust_cargo/workflow.py

```diff
@@ -0,0 +1,37 @@
+"""
+Rust Cargo Workflow
+"""
+import platform
+from aws_lambda_builders.path_resolver import PathResolver
+from aws_lambda_builders.workflow import BaseWorkflow, Capability
+from .actions import BuildAction, CopyAndRenameAction
+
+
+class RustCargoWorkflow(BaseWorkflow):
+
+    NAME = "RustCargoBuilder"
+
+    CAPABILITY = Capability(language="rust", dependency_manager="cargo", application_framework=None)
+
+    SUPPORTED_MANIFESTS = ["Cargo.toml"]
+
+    def __init__(self, source_dir, artifacts_dir, scratch_dir, manifest_path, runtime=None, mode=None, **kwargs):
+        super(RustCargoWorkflow, self).__init__(
+            source_dir, artifacts_dir, scratch_dir, manifest_path, runtime=runtime, **kwargs
+        )
+
+        # we utilize the handler identifier to
+        # select the binary to build
+        options = kwargs.get("options") or {}
+        handler = options.get("artifact_executable_name", None)
+        system_platform = platform.system().lower()
+        self.actions = [
```

Comment:
Is there any reason why we don't follow the 'scratch directory` concept the rest of the builders use?

Reply:
Could you provide some direction? These seems like it would not provide addition benefit for rust and further complicated the workflow implementation. 

The way rust handles build isolation is it's builtin notion of a target directory. The deployable components of that are copied to Sams notion of a target dir, the one provided as an argument to the workflow.